server/unleash.ts

```diff
@@ -47,5 +47,9 @@ export const getToggles = (veilederId, enhetId) => {
       "isOppfolgingISenFaseEnabled",
       context
     ),
+    isSenFaseFlexjarEnabled: unleash.isEnabled(
+      "isSenFaseFlexjarEnabled",
+      context
+    ),
```

Comment:
Hvor mange prosent tenker dere å slå på for?

Reply:
Vi tenkte å slå på for så få som mulig i starten for se hva vi får inn. Og basert på svarene kan vi skru opp eller omformulere spørsmålene for å gjøre flexjaren så nyttig som mulig.